### DISCUSSION TITLE
Cannot update to QField 2.0 on Motorola G7

### DISCUSSION THREAD

**Discussion by GDM-Tuebingen** [opengisch/QField#2779] [Q&A]
Hi there,

I used Qfield some time ago with Sync.
Now I use QFieldCloud. The project is on the cloud, but I can't use the App, because it's 1.10. Update fails, deinstall and install new brings only 1.10 not 2.0.
I use Motorola G4 with Android 7.0
Thanks for help!

**Comment by suricactus** [comment]
Hey, @GDM-Tuebingen, thanks for reaching out.

As we stated here https://www.opengis.ch/2022/04/05/qfield-2-0-is-here/#known-issues:

> Please note that on older AMRv7 architectures, some devices are suffering from a crash at launch. As such, we have not yet updated QField to 2.0 for these devices. If you own one such device and want to manually download and install QField, please visit the [release page on GitHub](https://qfield.org/releases).

**Reply by GDM-Tuebingen** [reply]
Hi suricactus,
thank you for the replay.
Seems that I need a new smartphone. With apk Installation 2.0 crashes immidiatly after starting. Before I can open a project.

**Reply by nirvn** [reply]
@GDM-Tuebingen , can you try the APKs here: https://github.com/opengisch/QField/pull/2773 -- the crashes on launch should hopefully be gone.

**Reply by nirvn** [reply]
@GDM-Tuebingen , assuming your device is an armv7, use the arm-android APK.
